### DISCUSSION TITLE
Possible to add average?

### DISCUSSION THREAD

**Discussion by fokcuk** [kalkih/mini-graph-card#710] [Q&A] [answered]
I have a utility meter that I am graphing daily consumption based on max value grouped by the day. Trying to get a rolling average of those maxes

**Comment by jlsjonas** [comment, marked as answer]
This is already possible; however, there's an open issue with average in your specific scenario (https://github.com/kalkih/mini-graph-card/issues/618)

**Reply by fokcuk** [reply]
Thank you!
